src/plugins/tables/tables-en.hbs

```diff
@@ -970,6 +970,127 @@
 			</tr>
 		</tbody>
 	</table>
+
+	<p>Adding the pagination at the top of the table and keeping it at the bottom</p>
```

Comment:
Please add a subheading to have a clearer separation between both working example.

```suggestion
	<h4>With top pagination example</h4>
	<p>Adding the pagination at the top of the table and keeping it at the bottom by using the following configuration: <code>data-wb-tables='{"ordering" : false, "dom":"&lt;\u0027top\u0027pfil>rt&lt;\u0027bottom\u0027p>&lt;\u0027clear\u0027>"}'</code></p>
```